content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+
+{{< img src="account_management/rbac/logs_write_exclusion_filters.png" alt="Grant write access on index exclusion filters to specific roles"  style="width:75%;" >}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+[2]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_pipelines
+
+Grants a role the ability to create and modify log processing pipelines. This includes setting matching filters for what logs should enter the processing pipeline, setting the name of the pipeline, and limiting which roles have write access on the processors within that pipeline (`logs_write_processors`).
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_write_pipeline_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+To grant write access to only two processing pipelines whose IDs are `abcd-1234` and `bcde-2345` respectively:
+
+1. Remove the global `logs_write_pipelines` permission on the role if already assigned.
+2. Get the UUID of the role you want to modify.
+3. Use the [Get Permission][2] API to find the `logs_write_pipelines` permission UUID for your region.
+4. Grant permission to that role with the following call:
+
+```sh
+curl -X POST \
+        https://app.datadoghq.com/api/v2/roles/<ROLE_UUID>/permissions \
+        -H "Content-Type: application/json" \
+        -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+        -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+        -d '{
+                "data": {
+                    "type": "permissions",
+                    "id": <PERMISSION_UUID>,
+                    "scope": {
+                        "pipelines": [
+                            "abcd-1234",
+                            "bcde-2345"
+                        ]
+                    }
+                }
+            }'
+```
+
+
+[1]: /api/#roles
+[2]: /api/#get-permissions
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_processors
+
+Grants a role the ability to create or modify the processors within a processing pipeline.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles page][1] and select the checkbox `write` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_write_processors_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+**Subset of Pipelines**:
+
+1. Remove the global permission on the role.
```

Comment:
in this case it means removing the `logs_write_processors` permission, but also making sure that the `logs_write_pipelines` permission is removed, since that inherently grants global processor edit permissions. 

I think it's worth making this explicit to avoid confusion

Reply:
Great points, would you mind making a suggestion?